clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1385,8 +1385,8 @@ public boolean matches(final AbstractRequest body) {
                 return body instanceof FetchRequest && ((FetchRequest) body).fetchData().containsKey(tp0);
             }
         }, fetchResponse(tp0, 1, 1), node);
-        time.sleep(heartbeatIntervalMs);
-        Thread.sleep(heartbeatIntervalMs);
+        time.sleep(heartbeatIntervalMs * 3);
```

Comment:
Why is this change needed ?

Reply:
Oh, I was tinkering with the test. Its not a neccesary change ... forgot to remove it.